src/Namotion.Reflection/XmlDocsExtensions.cs

```diff
@@ -972,7 +988,7 @@ private static string NormalizeTypeName(string typeName)
                 return file;
             }
 #endif
-            return GetXmlAssemblyFilePathForWindows(assembly, assemblyName);
+            return null;
         }
 
         private static string GetXmlAssemblyFilePathForWindows(dynamic? assembly, AssemblyName assemblyName)
```

Comment:
This method is not being used anymore...

Reply:
Removed.